### PR TITLE
Fix ime backend

### PR DESCRIPTION
**Pull request by jyvet**
Fix compilation of aiori backends requiring external libraries and add support for new function calls in the IME backend.

### CODE REVIEW

File: src/Makefile.am [hunk 1]
```diff
@@ -83,33 +82,35 @@ extraLDADD    += -laws4c
...
 
 MDTEST_SOURCES  = $(mdtest_SOURCES)
+MDTEST_LDFLAGS  = $(mdtest_LDFLAGS)
 MDTEST_LDADD    = $(mdtest_LDADD)
 MDTEST_CPPFLAGS = $(mdtest_CPPFLAGS)
 
 libaiori_a_SOURCES += $(extraSOURCES)
-#libaiori_a_LIBADD += $(extraLDFLAGS) $(extraLDADD)
-
+libaiori_a_LIBADD   =
```
Comment (sydidelot) [inline src/Makefile.am]: Can we remove this variable if it's not set?

File: src/aiori-IME.c [hunk 2]
```diff
@@ -228,9 +257,64 @@ static void IME_Delete(char *testFileName, IOR_param_t *param)
 /*
  * Determine API version.
  */
-static void IME_SetVersion(IOR_param_t *test)
+static char *IME_GetVersion()
+{
+        static char ver[1024] = {};
+#if (IME_NATIVE_API_VERSION >= 120)
```
Comment (sydidelot) [inline src/aiori-IME.c:263]: What will happen if `IME_NATIVE_API_VERSION` is not defined? Will it still compile?

File: src/Makefile.am (hunk 1 above)
Comment (jyvet) [reply src/Makefile.am]: I'll do that yes.

File: src/aiori-IME.c (hunk 2 above)
Comment (jyvet) [reply src/aiori-IME.c:263]: yes, this is equivalent to: 
`#if defined IME_NATIVE_API_VERSION && if IME_NATIVE_API_VERSION >= 120`
(see https://gcc.gnu.org/onlinedocs/gcc-3.0.2/cpp_4.html#SEC39)

Comment (sydidelot) [reply src/aiori-IME.c:263]: Ah, thanks :-)

**Review by sydidelot — APPROVED** [review]

**Pull request merged**